pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManagerTest.java

```diff
@@ -846,6 +858,39 @@ private String makeFakeSegmentName(int id) {
   }
 
   @Test
+  public void testCommitSegmentFile() throws Exception {
+    PinotLLCRealtimeSegmentManager realtimeSegmentManager = new FakePinotLLCRealtimeSegmentManager(false, Collections.<String>emptyList());
+    String tableName = "fakeTable_REALTIME";
+    String temporarySegmentName = "segment.tmp.123";
```

Comment:
use the getprefix method here. Not hard-code the file ame, but call yoiur new method to get the file name with uuid.

also, not temporarySegmentName